browser/resources/settings/brave_wallet_page/brave_wallet_page.ts

```diff
@@ -93,6 +98,13 @@ class SettingsBraveWalletPage extends SettingsBraveWalletPageBase {
     this.browserProxy_.isNftPinningEnabled().then(val => {
       this.isNftPinningEnabled_ = val
     });
+    this.browserProxy_.isTransactionSimulationsFeatureEnabled().then(val => {
+      this.isTransactionSimulationsFeatureEnabled = val
+    });
+    this.browserProxy_.getTransactionSimulationOptInStatusOptions()
+      .then(list => {
+        this.transaction_simulation_opt_in_options_ = JSON.parse(list)
```

Comment:
Why do we have to use a JSON list and parse it? Let's reference how we do getSolanaProviderOptions instead.

Reply:
I would also add if this options list is supposed to be static(i.e. doesn't change at runtime based on some backend's state) then let's just consider hardcoding these options here like for `cryptocurrency_list_` below

Reply:
The options will be static, but should implement it like `getSolanaProviderOptions` to ensure the enum values are correct in the UI?

Reply:
updated: aded764

Reply:
FWIW, I do agree with @supermassive here, but I don't hold a strong opinion here to block the PR.